docs/preset-env.md

```diff
@@ -73,6 +73,12 @@ or
 "browserslist": "> 0.25%, not dead"
 ```
 
+## Integrating with Webpack
```

Comment:
```suggestion
## Integrating with webpack
```

webpack is meant to be lowercase: https://webpack.js.org/contribute/writers-guide/#typesetting